pages/index.js

```diff
@@ -1,11 +1,14 @@
 import Heading from "../components/Heading";
 import SubHeading from "../components/SubHeading";
+import DogList from "../components/DogList";
 
 export default function Home() {
   return (
     <main>
-      <Heading>🐬 Capstone Template 🐬</Heading>
-      <SubHeading>We dolphinitely love coding!</SubHeading>
+      <Heading>Dog Pass 🐕‍🦺</Heading>
+      <SubHeading></SubHeading>
```

Comment:
You should delete the `<SubHeading></SubHeading>` line here, since you don't use it, or? You should remove the whole `SubHeading` component in the components folder, when you don't need it, or?